Plugin/external/ThirdParty/Deploy/Windows/Alembic/include/Alembic/Abc/All.h

```diff
@@ -0,0 +1,76 @@
+//-*****************************************************************************
```

Comment:
(a) kind of tragic we can't install all three platforms in the same place (and thus share the .h files)

(b) why is this in Windows/Alembic/include rather than in Windows/include? The cmake assumes you install all the dependences in a common prefix location.

Reply:
I did the easiest thing possible to get the abci.sln building. This is the way External.7z is organized - I'm unsure on the which actual versions of openexr, ilmbase, alembic have been built.

more work needs to be done to make the platform structure common.